app/controllers/lab_tests_controller.rb

```diff
@@ -24,6 +24,8 @@ def show
   def new
     @lab_test = LabTest.new
     authorize @lab_test
+    set_biomarkers
```

Comment:
You don't need to call `set_biomarkers` here as it was already called on [before_action](https://github.com/Code-the-Dream-School/health-keeper/pull/23/commits/e0eee2ca2c66870e99e783e21369193ae4232b18#diff-f7fcfdbad10e32bb265e9ba7415ebc17190f6dbd815cca2caeed74a00af4b6efR5).